Jenkinsfile

```
@@ -1,8 +1,7 @@
-@Library('pipeline-library@pull/784/head') _
-
 pipeline {
     agent {
-        label 'jdk17'
+        // infra.ci build on arm64 in the Dockerfile, as it's used in production
+        label 'jdk17 || linux-large-arm64'
```

```suggestion
        label 'jdk17 || linux-arm64-docker'
```
Docker is needed, using a (large) VM in infra.ci.jenkins.io